### DISCUSSION TITLE
Autolink Literals does not recognise links that end with an underscore

### DISCUSSION THREAD

**Discussion by samiemad** [remarkjs/remark#758] [Q&A] [answered]
I noticed that Autolink Literals do not handle links with trailing underscores correctly. 
For example:
https://example.com?param=abc__
will only show <https://example.com?param=abc> as link with the last two underscores as normal text.

You can also try it here https://remarkjs.github.io/react-markdown/

I also noticed that this is also the way github handles these links. but AFAIK links with trailing underscores are also valid. 
Is there a reason for this behavior? is it possible to change it?

**Comment by wooorm** [comment, marked as answer]
I’m not sure what the question is? remark-gfm follows GFM, it’s in the name. 🤷‍♂️

Put URLs in angle braces `<https://like-so>` or use a resource link `[whatever](like-so)` if you want underscores.
